src/main/java/com/ibm/cloud/sdk/core/http/Response.java

```diff
@@ -21,10 +21,12 @@
 
   private T result;
   private Headers headers;
+  private int statusCode;
 
   public Response(T result, okhttp3.Response httpResponse) {
     this.result = result;
     this.headers = new Headers(httpResponse.headers());
+    this.statusCode = httpResponse.code();
```

Comment:
While we're at it, would it also make sense to save off the httpResponse.message() value (status message)?

Reply:
The [spec](https://tools.ietf.org/html/rfc7230#section-3.1.2) calls that part a "reason-phrase"; what is your preferred naming here:
* `getReasonPhrase()`
* `getMessage()`
* `getStatusMessage()`
* something else?

Reply:
My initial response was going to be "statusMessage" because it will be somewhat in sync with "statusCode" and they are certainly related, but alas the Http spec braintrust has foiled our naming attempts...  Since the spec calls it a reason phrase, I guess we should do that as well:  `getReasonPhrase()`

Reply:
Talked with Rich on slack a bit and we convinced ourselves that getStatusMesage() would be more intuitive to users since it goes along with the status code.

Reply:
Added `getStatusMessage()` in c7c0b6a.